SystemTextJson.JsonDiffPatch.sln

```diff
@@ -0,0 +1,138 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Why moved solution file here?

Reply:
I thought it makes more sense to have the sln file at the root, since the src folder will contain the source for the library itself, and the test folder will contain the source code for the tests.

IMO, it's easier to find the sln file in the root, since many open source .NET repos follow this paradigm, and it makes more sense to reference like `src\SystemTextJson.JsonDiffPatch\SystemTextJson.JsonDiffPatch.csproj` & `test\SystemTextJson.JsonDiffPatch.UnitTests\SystemTextJson.JsonDiffPatch.UnitTests.csproj` rather than `SystemTextJson.JsonDiffPatch\SystemTextJson.JsonDiffPatch.csproj` & `..\test\SystemTextJson.JsonDiffPatch.UnitTests\SystemTextJson.JsonDiffPatch.UnitTests.csproj`.